src/main/scala/org/allenai/plugins/DeployPlugin.scala

```diff
@@ -144,7 +140,25 @@ object DeployPlugin extends AutoPlugin {
     IO.write(file, contents)
     log.info(s"Wrote ${contents.size} characters to ${file.getPath}.")
 
-    Seq(file)
+    file
+  }
+
+  lazy val npmBuildTask = Def.taskDyn {
+    if ((nodeEnv in thisProject).value == "dev") {
+      Def.task {
+        NodeJsPlugin.execBuild(
+          (NodeKeys.nodeProjectDir in Npm).value,
+          NodeJsPlugin.getEnvironment("dev", (NodeKeys.nodeProjectTarget in Npm).value)
```

Comment:
I assume the second param to execBuild is the environment variables? (I'll check below just noting my assumption)